ui.apps/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/README.md

```diff
@@ -151,6 +151,47 @@ You can also use dynamic references to access other fields' values.
 In order to define a reference, referenced field's name should be specified in dependsOnRef attribute.
 Then reference will be accessible in the query using @ or @@ symbol and reference name. 
 
+###### Using semicolon in DependsOn query 
+
+DependsOn Query is considered as single expression. 
```

Comment:
I would put it this way:
`DependsOn query is always treated as a single JavaScript expression, and never as multiple statements in one line. Semicolon symbols (";") within a DependsOn query must be escaped. If you add a query via a Java annotation semicolons escaped automatically..... But if you write directly to XML or HTML you should escape them by hand.....
If you actually need to execute several JavaScript statements within a DependsOn, you can do this by wrapping them in a function call..... Mind that it is still better to move complex structures to a standaline client library`